src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -277,70 +255,28 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
     const status = getQueryStatus(query.status);
 
     return (
-      <DocumentTitle title={`Alerts- ${orgId} - Sentry`}>
+      <DocumentTitle title={`Alerts - ${orgId} - Sentry`}>
```

Comment:
`<SentryDocumentTitle>` might be preferred here so you don't have to worry about the formatting and can add i18n
```suggestion
      <SentryDocumentTitle title={t('Alerts')} objSlug={organization.slug}>
```